kits/democracy/test/democracy.js

```diff
@@ -72,10 +72,10 @@ contract('Democracy Kit', accounts => {
         const networkName = (await getNetwork(networks)).name
         if (networkName == 'devnet' || networkName == 'rpc') {
             // transfer some ETH to other accounts
-            await web3.eth.sendTransaction({ from: owner, to: holder20, value: web3.toWei(1, 'ether') })
-            await web3.eth.sendTransaction({ from: owner, to: holder29, value: web3.toWei(1, 'ether') })
-            await web3.eth.sendTransaction({ from: owner, to: holder51, value: web3.toWei(1, 'ether') })
-            await web3.eth.sendTransaction({ from: owner, to: nonHolder, value: web3.toWei(1, 'ether') })
+            await web3.eth.sendTransaction({ from: owner, to: holder20, value: web3.toWei(10, 'ether') })
+            await web3.eth.sendTransaction({ from: owner, to: holder29, value: web3.toWei(10, 'ether') })
+            await web3.eth.sendTransaction({ from: owner, to: holder51, value: web3.toWei(10, 'ether') })
+            await web3.eth.sendTransaction({ from: owner, to: nonHolder, value: web3.toWei(10, 'ether') })
```

Comment:
😄

Reply:
Haha, yes, so much time wasted for such an easy solution. Anyway, I still think there's something weird going on here, because it was randomly failing and because if I'm not wrong the transaction which failed was casting a vote and the account performing it had 0.6 ETH (don't know what gas price we have at that node but it should be quite high).